### ISSUE TITLE
[Skyrim]ESPs not getting loaded 

### ISSUE THREAD

**Issue by Ufeldraku** [Modorganizer2/modorganizer#332]
Problem:
MO2 doesnt load the esps, only the esps, to the game.
LOOT, tes5edit, any other tool run through it can see the esps
Files like sound and textures do work on the game.

* Mod Organizer Version that exhibits the issue: 2.1.2
* Last Mod Organizer Verison that did not exhibit the issue (if applicable): 2.1.2, ironically
* Desktop OS/version used to run Mod Organizer: Windows 10 1709 64 bit.

# Details

Basically the same thing to what problem describes.
Also, Neither of the 3 dlcs gets loaded.
# Link to Mod Organizer logs

## USVFS
https://gist.github.com/Ufeldraku/99e7fa1e60508c60b53066dd8c10b795

## MO Interface
https://gist.github.com/Ufeldraku/52839b74c89bf9235b8f077b36ef22c5

**Comment by Ufeldraku** [comment]
I am the guy from discord, so I am going to close this issue. If I have problems after I buy the game, I will report back.

**Comment by Silarn** [comment]
I'm going to assume this is the same report that was on discord.
